src/main/resources/BaseTemplate.json

```diff
@@ -0,0 +1,202 @@
+{
+  "Parameters": {
+    "SubnetId": {
+      "Type": "AWS::EC2::Subnet::Id"
+    },
+    "AMIId": {
+      "Type": "String",
+      "Default": "ami-062f7200baf2fa504"
+    },
+    "InstanceType": {
+      "Type": "String",
+      "Default": "t2.medium"
```

Comment:
Just a `Default` value, but AWS regions launched after `t3.` generation instance types launched in 2018 don't have older `t2.` generation instance types